src/main/java/org/codebusters/audiogeek/preferencesagent/application/rest/mygenres/GetMyGenresResponse.java

```diff
@@ -0,0 +1,6 @@
+package org.codebusters.audiogeek.preferencesagent.application.rest.mygenres;
+
+import java.util.Set;
+
+public record GetMyGenresResponse(Set<String> genres) {
```

Comment:
Public? Really? Pls thing more carefully about access privileges. Check all classes in your PR.